prometheus-pusher/main.go

```diff
@@ -110,9 +114,30 @@ func (p *prometheusSink) createPusher(jobName string) (*push.Pusher, error) {
 	return pusher, nil
 }
 
+func parseStringToMap(envValue string) map[string]string {
+	items := make(map[string]string)
+	if envValue == "" {
+		return items
+	}
+	datas := strings.Split(envValue, ",")
+	getKeyVal := func(item string) (key, val string) {
+		splits := strings.Split(item, "=")
+		key = splits[0]
+		val = splits[1]
```

Comment:
will this panic if split doesn't create 2 elements?

Reply:
good catch